examples/options/src/options/common_options.cpp

```diff
@@ -28,8 +28,10 @@ traccc::common_options::common_options(po::options_description& desc) {
                        "generate performance result");
     desc.add_options()("detector_file", po::value<std::string>()->required(),
                        "specify detector file");
-    desc.add_options()("material_file", po::value<std::string>(),
+    desc.add_options()("material_file", po::value<std::string>()->required(),
                        "specify material file");
+    desc.add_options()("grid_file", po::value<std::string>()->required(),
```

Comment:
Are material and grids really required for every algorithm? If for e.g. seeding you just need the geometry, then why go through the entire grid building process?

Reply:
I know the names are a bit misleading but every example has a fitting algorithm at least. For example, seeding_example is made of seeding, finding and fitting.